src/components/common/NavTabs/index.tsx

```diff
@@ -32,7 +32,7 @@ const NextLinkComposed = forwardRef<HTMLAnchorElement, Props>(function NextCompo
 
 const NavTabs = ({ tabs }: { tabs: NavItem[] }) => {
   const router = useRouter()
-  const activeTab = tabs.map((tab) => tab.href).indexOf(router.pathname)
+  const activeTab = Math.max(0, tabs.map((tab) => tab.href).indexOf(router.pathname))
```

Comment:
Why did we need to change this? If the user is not on one of the `tabs`, I don't think we should default to one of them.

Reply:
Because it was throwing lots of errors about a -1 tab.

Reply:
Ok, makes sense then.